src/scripts/zotonic-runtests

```diff
@@ -55,7 +55,7 @@ case "$1" in
         echo
 
         # Run the tests
-        $ERL +P 10000000 +K true -pa $PA $NAME_ARG ${NODENAME}_testsandbox@$NODEHOST -boot start_sasl -sasl errlog_type error -s zotonic -eval "timer:sleep(8000),init:stop(case eunit:test([$ALL],[]) of error -> 1; ok -> 0 end)"
+        $ERL +P 10000000 +K true -pa $PA $NAME_ARG ${NODENAME}_testsandbox@$NODEHOST -boot start_sasl -sasl errlog_type error -s zotonic -config /etc/zotonic/zotonic.config -eval "timer:sleep(8000),init:stop(case eunit:test([$ALL],[]) of error -> 1; ok -> 0 end)"
```

Comment:
TODO: remove hardcoded reference to config file. How do we find the user’s database credentials?